rules/azure-resources-creating/rule.md

```diff
@@ -21,14 +21,13 @@ created: 2020-10-06T00:13:27.000Z
 archivedreason: null
 guid: 007dd1f6-8ac6-4840-8f4f-a39c6f847880
 ---
-
 We've been down this road before where developers had to be taught not to manually create databases and tables. Now, in the cloud world, we're saying the same thing again:  **Don't manually create Azure resources.**
 
 <!--endintro-->
 
 ### Manually Creating Resources
 
-This is the most common and the worst. This is bad because it requires manual effort to reproduce and leaves margin for human error.
+This is the most common and the worst. This is bad because it requires manual effort to reproduce and leaves margin for human error. Manually provisioning resources can also lead to [environment drift](https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/overview?tabs=json), which is to say that over time it can be difficult to keep track of which deployment configurations were made and why.
```

Comment:
Hi Caleb.

That link doesn't seem to point to anything defining environment drift. Also, more often we use the phrase 'configuration drift' instead of 'environment drift' - maybe see if you can find a page that talks specifically about the one you are thinking of.

Reply:
I think I may have used the wrong link when I modified this rule, so I've updated it to point to the correct microsoft learn article. I've also updated the phrasing to 'configuration drift'. 

Please let me know what you think